### PR TITLE
Added automation-bundle

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR https://github.com/sulu/sulu/pull/3037

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -34,8 +34,17 @@
         "doctrine/doctrine-fixtures-bundle": "2.2.*",
         "doctrine/doctrine-bundle": "^1.0",
         "doctrine/doctrine-cache-bundle": "^1.0",
-        "oro/doctrine-extensions": "^1.0"
+        "oro/doctrine-extensions": "^1.0",
+
+        "php-task/task-bundle": "@dev",
+        "php-task/php-admitask": "@dev"
```
Comment (danrot) [inline composer.json]: That's an error, right?

**Review by danrot — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (wachterjohannes) [reply composer.json]: yes thats an error (:

**Comment by wachterjohannes** [comment]
@danrot fixed

**Comment by danrot** [comment]
Please update all the dependencies to a stable one, and create a corresponding PR for sulu-standard.

**Comment by wachterjohannes** [comment]
@danrot updated sulu dependency to develop

**Review by danrot — APPROVED** [review]

**Pull request merged**